src/cljs/main/broadfcui/page/workspace/summary/catalog.cljs

```diff
@@ -91,128 +94,110 @@
+                 {:keys [type enum consentCode renderHint] :as prop} (get-in library-schema [:properties property])
+                 {:keys [wording datatype emptyChoice]} renderHint
+                 required? (contains? required-attributes property)
+                 error? (contains? invalid-properties property)
+                 colorize #(merge % (when error? {:borderColor (:exception-state style/colors)
+                                                  :color (:exception-state style/colors)}))
+                 update-property #(swap! state update :attributes assoc property (.. % -target -value))
+                 checkbox (fn [{:keys [val label]}]
+                            [:label {:style (colorize {:display "inline-flex" :alignItems "center"
+                                                       :cursor "pointer" :marginRight "2em"})}
+                             [:input (merge
+                                      {:type "radio" :style {:cursor "pointer"}
+                                       :onClick #(swap! state update :attributes assoc property val)}
+                                      (when (= val current-value) {:checked true}))]
+                             [:div {:style {:padding "0 0.4em" :fontWeight "500"}} (or label (str val))]])]
+             (when-not (:hidden prop)
+               [(if enumerate :li :div) {}
+                [:div {:style {:marginBottom 2}}
+                 (:title prop)
+                 (when consentCode
+                   (list
+                    " ["
+                    [:abbr {:style {:cursor "help" :whiteSpace "nowrap" :borderBottom "1px dotted"}
+                            :title (get-in library-schema [:consentCodes (keyword consentCode)])}
+                     consentCode]
+                    "]"))
+                 (when required?
+                   [:span {:style (colorize {:fontWeight "bold"})}
+                    " (required)"])]
+                (cond enum
+                      (if (< (count enum) 4)
+                        [:div {:style {:display "inline-block" :margin "0.75em 0 0.75em 1em"}}
+                         (map #(checkbox {:val %}) enum)]
+                        (style/create-identity-select {:value (or current-value ENUM_EMPTY_CHOICE)
+                                                       :style (colorize {})
+                                                       :onChange update-property}
+                                                      (cons ENUM_EMPTY_CHOICE enum)))
+                      (= type "boolean")
+                      [:div {:style {:display "inline-block" :margin "0.75em 0 0.75em 1em"}}
+                       (checkbox {:val true :label (case wording "yes/no" "Yes" "True")})
+                       (checkbox {:val false :label (case wording "yes/no" "No" "False")})
+                       (when-not required?
+                         (checkbox {:val nil :label (or emptyChoice "N/A")}))]
```

Comment:
see my comment on the orch PR - I believe Moran said there's a difference between (required + yes/no/unspecified) and not-required. If that's true, will have to redo this code.